api/apps/api/test/access-control/project-acl.fixtures.ts

```diff
@@ -0,0 +1,242 @@
+import * as request from 'supertest';
```

Comment:
Seems that fixtures/tests are trying to test too many things within one place

Reply:
Same as https://github.com/Vizzuality/marxan-cloud/pull/640#discussion_r757509916